changelog.txt

```diff
@@ -1,5 +1,8 @@
 v.Next
 ----------
+
+Version 4.2.0
+-------------
```

Comment:
Could you also add the common version 4.2.0 uses in the change log
something like.

- [PATCH] update common@7.0.0